.github/workflows/ci.yaml

```diff
@@ -25,8 +25,7 @@ jobs:
       run: |
         go install github.com/onsi/ginkgo/ginkgo
         go get -u golang.org/x/lint/golint
-        curl -LO https://github.com/dominikh/go-tools/releases/download/2020.1.3/staticcheck_linux_amd64.tar.gz
-        sudo tar xfz staticcheck_linux_amd64.tar.gz --strip-component 1 -C $GOPATH/bin staticcheck/staticcheck
+        go get honnef.co/go/tools/cmd/staticcheck
```

Comment:
Hm, is that necessary? I thought it's safer if we use a fixed version of staticcheck?

Reply:
Retracted to curl solution.